### ISSUE TITLE
New terms to distinguish additional URLs in a dataset

### ISSUE THREAD

**Issue by agarny** [SciCrunch/sparc-curation#74]
**Background:** we have a [(simulation) dataset](https://app.pennsieve.io/N:organization:618e8dd9-f8d2-4dc4-9abb-c6aaab2e78a0/datasets/N:dataset:ab9a8fb2-b2a2-478c-aa90-f035b46d0eb9/overview) that contains some additional URLs (stored in [`dataset_description.xlsx`](https://github.com/SciCrunch/sparc-curation/files/6424402/dataset_description.xlsx) using the `Additional Links` field). To distinguish those URLs, we use the corresponding information stored in the `Link Description` field, i.e.
 - https://models.physiomeproject.org/workspace/486 ---> PMR workspace;
 - https://models.physiomeproject.org/workspace/486/rawfile/55879cbc485e2d4c41f3dc6d60424b849f94c4ee/HumanSAN_Fabbri_Fantini_Wilders_Severi_2017.cellml ---> CellML file; and
 - https://models.physiomeproject.org/workspace/486/rawfile/55879cbc485e2d4c41f3dc6d60424b849f94c4ee/HumanSAN_Fabbri_Fantini_Wilders_Severi_2017.rdf ---> RDF file.

**Request:** the above might be fine, but the information available in the `Link Description` field should be formalised and we would like to suggest the following terms:
 - `PMR workspace`;
 - `CellML file`;
 - `SED-ML file`;
 - `COMBINE archive`; and
 - `RDF file`.

@nickerso, would you agree with those terms?

**Comment by nickerso** [comment]
Only suggestion might be `repository` rather than `PMR workspace`.

@tgbugs - are these the datacite relationship types that you mentioned are part of the SDS 2.0: https://support.datacite.org/docs/relationtype_for_citation. And if so, these terms would be used to identify the relationship between the dataset and the resource being linked to? (rather than the current set which are more like mime-types)

**Comment by agarny** [comment]
Wouldn't `repository` (or `Repository`) be better suited to reference a whole repository as such (e.g., https://models.physiomeproject.org/)?

**Comment by nickerso** [comment]
a workspace is a standalone (git) repository - but you could also imagine a repository on, for example, github or biomodels. With this approach there is no technical restriction to only having models from PMR...but there is likely a curation restriction to having models from known reliable and permanent sources.

**Comment by agarny** [comment]
Oh, ok, repository as in Git repository, as opposed to Physiome Model Repository. Fair enough.

(I have updated my simulation dataset and now use `Repository` rather than `PMR workspace` for https://models.physiomeproject.org/workspace/486.)

**Comment by tgbugs** [comment]
@nickerso yes, those are the relation types I was referring to. They function as you describe. There are also resource types and resource type general which function like the mimetypes that are used in the manifest file `additional types` column.
